tox.ini

```diff
@@ -28,9 +27,7 @@ deps =
     pytest73: pytest==7.3.*
     pytest74: pytest==7.4.*
```

Comment:
I would already add a placeholder for the 8.0.x series (with a temporary `pre` option), given these changes affect that branch and currently, we don't have an idea whether all is fine, or it is only passing with 8.1.dev